invisible_cities/reco/monitor_functions_test.py

```diff
@@ -417,7 +406,7 @@ def test_kdst_bins():
 
 
 @given(data_frames(columns=columns(kdst_variables, elements=floats(allow_nan=False))))
-@settings(deadline=None)
+@settings(deadline=None, suppress_health_check=(HealthCheck.too_slow,))
```

Comment:
Maybe we can constrain the  index range? 
`index=range_indexes(min_size=1, max_size=10)`
But I agree that it still requires HealrCheck to be suppressed

Reply:
Sounds good